src/main/java/com/napier/sem/UserPrompt.java

```diff
@@ -0,0 +1,318 @@
+package com.napier.sem;
+
+import java.util.*;
+
+public class UserPrompt
+{
+    private final DAO dao;
+    private final Scanner scanner;
+    boolean userWantsToQuit;
+    private static final int[] topNQueryIds = new int[]{2, 4, 6};
+    private static final String INITIAL_QUERY_PROMPT = App.HORIZONTAL_LINE + "\n" +
+            "Enter the number corresponding to the type of query you'd like to run\n" +
+            "Your options are as follows:\n\n" +
+            "1) All countries in...\n" +
+            "2) Top N countries in...\n" +
+            "3) All cities in...\n" +
+            "4) Top N cities in...\n" +
+            "5) All capital cities in...\n" +
+            "6) Top N capital cities in...\n" +
+            "7) Residence report of people living in...\n" +
+            "8) Population of...\n" +
+            "9) Report on number of speakers per language for Chinese, English, Hindi, Spanish and Arabic\n\n" +
+            "All results will be sorted in order of largest population to smallest\n" +
+            "Enter 'q' at any time to exit";
+    private final HashMap<Integer, QueryInfo> queryTable = new HashMap<>();
```

Comment:
a hashmap is an object like a list, but it has key/value pairs. in our case it looks like 
```
queryTable = { 1: QueryInfo object, 2: QueryInfo object . . . }
```
where the key is the queryId (1) and the value is the object with the information about that query